packages/eslint-plugin-wpcalypso/package.json

```diff
@@ -20,5 +20,12 @@
 	"engines": {
 		"node": ">=10"
 	},
+	"peerDependencies": {
+		"eslint": "^3.3.1 || ^4.6.1 || ^5.0.0",
+		"eslint-plugin-wpcalypso": "^3.4.1 || ^4.0.0"
```

Comment:
I don't think the module should depend on itself 🙂 

```suggestion
```

Reply:
Well spotted! 😆